roaringbitmap/src/main/java/org/roaringbitmap/RunContainer.java

```diff
@@ -808,7 +808,7 @@ protected boolean contains(ArrayContainer arrayContainer) {
       return false;
     }
     int ia = 0, ir = 0;
-    while(ia < arrayContainer.getCardinality() && ir <= runCount) {
+    while(ia < arrayContainer.getCardinality() && ir < runCount) {
       int start = getValue(ir);
       int stop = start + toIntUnsigned(getLength(ir));
       if(arrayContainer.content[ia] < start) {
```

Comment:
There's another one here; we should be storing `arrayContainer.content[ia]` in an int variable:

```java
int value = Util.toIntUnsigned(arrayContainer.content[ia]);
if (value < start) {
...
} else if (value > stop) {
...```